cvmfs/download.cc

```diff
@@ -177,25 +177,27 @@ static size_t CallbackCurlHeader(void *ptr, size_t size, size_t nmemb,
   //          header_line.c_str());
 
   // Check http status codes
-  if (HasPrefix(header_line, "HTTP/1.", false)) {
-    if (header_line.length() < 10)
+  if (HasPrefix(header_line, "HTTP/", false)) {
+    if (header_line.length() < 12) {
```

Comment:
"HTTP/1.1" -> 8 characters
"HTTP/2"    -> 6 characters
So two less. That's basically the reason. The rest is (should be?) equal